src/parser/parser.ts

```diff
@@ -10,8 +10,16 @@ import { globSync } from 'glob';
 export type Renderer = (content: string) => string;
 
 const pathHeading: Renderer = (path: string) => `# \`${path.replace(homedir(), '~')}\``;
-const wrap = (contentType: string, content: string) =>
-    `<div class="content-${contentType}">${content}</div>`;
+
+function wrap(contentType: string, content: string, linkPath?: string): string {
+    let link = '';
+    if (typeof linkPath !== 'undefined') {
+        link = `\n<div id="top-nav">\n<a href="${pathToURL(linkPath)}">◂ ${
+            pbasename(linkPath) || '/'
+        }</a>\n</div>`;
```

Comment:
Actually, let's also omit the `◂ ` part here and instead handle it in the CSS, e.g. with something like

```css
#top-nav > a:before { content: '◂' }
```

This way people can easily customize and adjust that part

Reply:
Can do.

There is a chance this will change again when the .svg icons go in.

Reply:
For reference you can look at this

https://github.com/jannis-baum/Vivify/blob/ba76b52bb46f21b053d9c3dc9eeb2da50d909ced/static/style.css#L106-L108

> There is a chance this will change again when the .svg icons go in.

Yes, probably, I'd just do it like this now anyways. Maybe the SVG icons can also be added this way

Reply:
Yeah fair enough.

Actually I just realised its now 1:30 am and I do have work tomorrow.

I'll better head off and I'll do this in the morning.

Reply:
No rush! Sleep is definitely the priority ^^